pybaseball/playerid_lookup.py

```diff
@@ -84,6 +99,24 @@ def playerid_lookup(last=None, first=None, player_list=None):
 
     results = results.reset_index(drop=True)
 
+    if len(results) == 0:
```

Comment:
this seems complicated enough that it could go in its own function, `get_similar_matches` or something.

Reply:
simplified, now just a couple of lines